src/Microsoft.Identity.Web/IdHelper.cs

```diff
@@ -8,35 +8,35 @@
 
 namespace Microsoft.Identity.Web
 {
-    internal static class IdHelper
-    {
-        private static readonly Lazy<string> s_idWebVersion = new Lazy<string>(
-           () =>
-           {
+	internal static class IdHelper
+	{
+		private static readonly Lazy<string> s_idWebVersion = new Lazy<string>(
+		   () =>
+		   {
 #pragma warning disable CS8600 // Converting null literal or possible null value to non-nullable type.
-               string fullVersion = typeof(IdHelper).GetTypeInfo().Assembly.FullName;
+			   string fullVersion = typeof(IdHelper).GetTypeInfo().Assembly.FullName;
```

Comment:
Why is the FullName field parsed rather than using the .Version property of the Assembly object?